.github/workflows/run-tests.yml

```diff
@@ -45,6 +45,7 @@ jobs:
           -Dsonar.organization=tyktechnologies
           -Dsonar.projectKey=TykTechnologies_tyk-ui
           -Dsonar.sources=./src
+          -Dsonar.coverage.exclusions=./cypress/**/*.js,**/*.test.js,./src/form/components/Combobox/*.js,./src/form/components/redux-form/**/*.js
           -Dsonar.coverage.exclusions=**/*.test.js
```

Comment:
There are two lines for the same config property. The second one should be removed.